### ISSUE TITLE
Updates to requirements.txt Require 3 Builds

### ISSUE THREAD

**Issue by mesknock** [knockrentals/lambda-builder#16]
Currently, after modifying `requirements.txt` you must invoke `make build` 3 times in order to produce a ZIP file with the dependencies included.

This seems to be an issue with the order we are building the dependencies in. It might be as simple as updating one of the variables to be recursive in order to ensure the need dependency files are picked up as prerequisites for the ZIP files. I am thinking this is the case due to the fact that subsequent builds "see" the new dependency files and then move them accordingly.

**TLDR**: `make build build build` after updating `requirements.txt` until this issue is resolved.
